### PR TITLE
fix(clayui.com): new landing page proposal to fix accessibility issues

### LINKED ISSUE #3616: Accessibility issues in landing page
## What are the steps to reproduce?

Use Axe to validate Landing page accessibility

- Almost all errors are related to accessibility
- One more related to `ID` duplicity

image

- There are more accessibility errors not located by automatic validation, like not coherence in links, not homogeneity using `links icons` etc.

I've created a document:
- https://docs.google.com/document/d/1VFcq_E4JZHq7RO3vuyHcTu76xTEHmWq7dbr7zDEwd9A/edit?usp=sharing

There are some proposals for fixing issues by a quick redesign.

## What is the expected result?

Not have serious accessibility problems in the landing page

### PR DESCRIPTION
**Pull request by marcoscv-work**
This PR fixes #3616

- The last commit is only a cosmetic change

PR resend from: https://github.com/liferay/clay/pull/3617

### CODE REVIEW

**Comment by marcoscv-work** [comment]
image

File: clayui.com/src/components/LayoutNavHome/LayoutNavHome.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+					<div
+						className="d-inline-block text-decoration-none text-reset"
+					>
+						<span className="c-inner" tabIndex="-1">
+							<img
+								alt="Clay"
+								className="align-middle sidebar-logo-image"
+								height="32"
+								src="/images/clay_logo_w.png"
+							/>
```
Comment (kresimir-coko) [inline clayui.com/src/components/LayoutNavHome/LayoutNavHome.js:23]: I think that we like having an empty line separating sibling elements.

File: clayui.com/src/components/LayoutNavHome/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * SPDX-FileCopyrightText: © 2018 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import LayoutNavHome from './LayoutNavHome';
+
+export default LayoutNavHome;
```
Comment (kresimir-coko) [inline clayui.com/src/components/LayoutNavHome/index.js:8]: If you take a look at other similar components in `clayui.com/src/components` like `IconSearch` and `CodeToggle` you can see the exporting pattern that doesn't require 2 files: `LayoutNavHome.js` and `index.js` files but just the index file.

**Comment by marcoscv-work** [comment]
@kresimir-coko added the change! thanks!

**Comment by marcoscv-work** [comment]
Can we merge it then?

**Comment by matuzalemsteles** [comment]
@marcoscv-work yes, I will merge this. I realized now that @bryceosterhaus is in PTO.

**Comment by bryceosterhaus** [comment]
New changes look great! Thanks @marcoscv-work!

**Pull request merged**